README.md

```diff
@@ -1,32 +1,28 @@
 # Podman Desktop Red Hat Account Extension
 
-An extension for Podman Desktop to simplify logging into and creating a Red Hat account.  The extension opens sso.redhat.com in the browser to retrieve an SSO token upon successful login.  The SSO token is then used to log into the Red Hat Container Registry ([registry.redhat.io](https://catalog.redhat.com/)) and to register the Linux virtual machine powering Podman Desktop via subscription-manager to grant the containers access to protected Red Hat content such as RHEL repositories.  The SSO form guarantess that each user, new and existing, has a valid [Red Hat Developers Subscription](https://developers.redhat.com/) which will be used to register the virtual machine at no cost.
+An extension for Podman Desktop to simplify logging into and creating a Red Hat account.  The extension opens sso.redhat.com in the browser to retrieve an SSO token upon successful login.  The SSO token is then used to log into the Red Hat Container Registry ([registry.redhat.io](https://catalog.redhat.com/)) and to register the Linux virtual machine powering Podman Desktop via subscription-manager to grant the containers access to protected Red Hat content such as RHEL repositories.  The SSO form guarantees that each user, new and existing, has a valid [Red Hat Developers Subscription](https://developers.redhat.com/) which will be used to register the virtual machine at no cost.
 
 # Installation
 
-The extension is currently available in an Alpha version but ready to test.  Extension for Podman Desktop are shipped as OCI container images.  Please refer to the [Podman Desktop documentation](https://podman-desktop.io/docs/extensions/install) for installation instructions and to [Quay.io](https://quay.io/repository/redhat-developer/podman-desktop-redhat-account-ext?tab=tags) for available tags.
+You can install the Red Account Extension directly inside of Podman Desktop.  It is also part of the Red Hat Extension Pack:
 
-The latest available image is `quay.io/redhat-developer/podman-desktop-redhat-account-ext:0.0.2`.
+![image](https://raw.githubusercontent.com/redhat-developer/podman-desktop-redhat-account-ext/main/screenshots/extension-pack.png)
```

Comment:
Why don't we use relative URLs ? That would allow to compare between versions ?

Reply:
@dgolovin and @benoitf don't like relative URLs.